doc/src/documentation/architecture/adr/adr006-kiss-frontend-components.md

```diff
@@ -0,0 +1,25 @@
+# ADR-006 frontend components without complex logic
+
+## Status
+
+<adr-status status='accepted' />
+
+## Context
+
+By components are meant here vue components, in the form of single-file components. These contain a template,
```

Comment:
The term components stands for the vue components, in the form of single-file components, in this context.